docs/user/builds.rst

```diff
@@ -68,6 +68,46 @@ The following are the pre-defined jobs executed by Read the Docs:
     it's possible to run user-defined commands and :doc:`customize the build process <build-customization>`.
 
 
+Cancelling builds
+-----------------
+
+There may be situations where you want to cancel a particular running build.
+Cancelling running builds will allow your team to speed up review times and also help us reduce server costs and ultimately,
+our environmental footprint.
+
+Consider the following scenarios:
+
+* the build depends on an external situation that's not met yet
+* there were no changes on the documentation files
+* the user wants it for any particular reason
```

Comment:
This is not clear to me? Not 100% sure what it's trying to say, but we can be more explicit here, or not include it if it's just "other"

Reply:
My idea here was to communicate the developer could have "any reason" and know upfront they want to skip the build for this particular push/commit. Because of that "any reason" the user will add `skip ci` in the commit message.

I'm not sure how to explain that situation in a better way (without mentioning the commit message, since that's the implementation, not the scenario)

Reply:
```suggestion
* many other use cases that can be solved with custom logic
```